### ISSUE TITLE
Feature request: package entry point for reactive core

### ISSUE THREAD

**Issue by vveisard** [solidjs/solid#2088]
The current package entry points use conditional exports which make assumptions about my desired behavior, and provide different implementations based on my environment. In my case, I want to use solid as a reactive state management library on my server, and assumed I wanted SSR with no reactivity. 

I want an entry point like `solid-js/reactive` to import the reactive core of solid-js (signals, effects, memos, etc) without any assumptions being made.

**Comment by ryansolid** [comment]
Yes.. the plan is to export the reactive core separately in our next major version.
